activities.json

```diff
@@ -696,16 +696,16 @@
     "url": "https://w3c.github.io/mst-content-hint/"
   },
   {
-    "ciuName": "native-filesystem-api",
-    "description": "This document defines a web platform API that lets websites gain write access to the native file system. It builds on [FILE-API], but adds lots of new functionality on top.",
-    "id": "native-file-system",
+    "ciuName": "file-system-access-api",
+    "description": "This document defines a web platform API that lets websites gain write access to the local file system. It builds on [FILE-API], but adds lots of new functionality on top.",
```

Comment:
Change [FILE-API] to File API while you're here?

Reply:
Done